### ISSUE TITLE
Kamodo won't accept units 'N', 'S', or 'Ohm'

### ISSUE THREAD

**Issue by rebeccaringuette** [nasa/Kamodo#34]
Inputting units 'N', 'S', 'Ohm', or any unit string with those units causes an error. I have a partial fix you can use from KamodofyPlasmaPy (https://github.com/rebeccaringuette/KamodofyPlasmaPy.git), which is copied below. This function is used in the call to kamodofy, and I have had no problems with the function representation or execution in kamodo. An example call is:
```py
kamodo['SecondLaw'] = kamodofy(func, units=KamodoUnit('N'), arg_units={'m':KamodoUnit('kg'),'a':KamodoUnit('m/s**2')} 
```
The important piece of code is:
```py
def KamodoUnit(old_unit):  #takes any string, expects it to represent units
    for item in ['S','N','Ohm']:  #force sympy to accept by decomposing the unit
        if item in old_unit: old_unit = u.Unit(old_unit).decompose().to_string('ogip')
    unit_list = [(' / ','/'),(' * ','*'),(' ','*')]  #cannot accept spaces from astropy
    for old, new in unit_list: old_unit = old_unit.replace(old,new)
    if 'radian' not in old_unit: old_unit = old_unit.replace('rad','radian')
    return old_unit
```
(where u is from astropy import units as u) I run all of my units through this code without issue. The hardcoding of the problematic units (lines 2 and 3 of the above section) can be changed to be 
```py
if not hasattr(parse_expr(unit_str),'subs'):  #if not in sympy dictionary, decompose
    old_unit = u.Unit(old_unit).decompose().to_string('ogip')
```
Interestingly, sympy already has the units I had to hardcode this for (newtons, siemens, ohms), but won't accept them through kamodo. This issue may resolve itself if this other problem can be fixed.

**Comment by asherp** [comment]
This is fixed in my branch. You should be able to run the following

```python
from kamodo import get_unit

get_unit('ohm')
get_unit('S')
get_unit('N')
```
https://github.com/asherp/Kamodo/commit/84b17302a91115a5133502cc170a468380833e22

**Comment by rebeccaringuette** [comment]
Cool! I'll download it and test it in my notebook by attempting to kamodofy
a function with those units.

Rebecca Ringuette

**Comment by rebeccaringuette** [comment]
I tried with the version on your branch, and it still doesn't work. The
code I executed was
```py
kamodo_test = Kamodo()
kamodo_test['F1[N]'] = kamodofy(lambda F=np.linspace(0.1,1,10): F,
units='N')
kamodo_test
```
I don't get the error anymore. Instead I get output this (assuming latex
notation):
```py
F_1(F)[<functionNat0x000001E179ECE828>] = \lambda(F)
```
Also, the error with the function signature returned. Any function from
plasmapy that has an argument list with all dimensionless units and a
return value with dimensionless units fails. As before, replacing line 271
with the code I suggested in the posted issue (#36) resolved the issue.
Thanks, Asher.

Rebecca Ringuette
